src/main/kotlin/uk/gov/dwp/dataworks/logging/LoggingUtils.kt

```diff
@@ -31,28 +31,29 @@ fun throwableProxyEventToJsonKeyPair(event: ILoggingEvent): String {
     val throwableProxy = event.throwableProxy ?: return ""
 
     val stackTrace = ThrowableProxyUtil.asString(throwableProxy)
-    return """"exception":"${StringEscapeUtils.escapeJson(flattenString(stackTrace))}", """
+    return "\"exception\":\"${StringEscapeUtils.escapeJson(flattenString(stackTrace))}\", "
 }
 
 /**
  * Converts a message and a set of Tuples to an _almost_ formatted Json string. Tuples are first parsed as follows:
  * ```
- *  "TupleKey1":"TupleValue1", "TupleKey2":"TupleValue2"
+ *  TupleKey1":"TupleValue1", "TupleKey2":"TupleValue2"
  * ```
  * Then the input message is escaped as per [StringEscapeUtils.escapeJson] and prepended with quotes. The resulting
  * output will look like the following:
  * ```
- *  "input message contents", "TupleKey1":"TupleValue1", "TupleKey2":"TupleValue2"
+ *  input message contents", "TupleKey1":"TupleValue1", "TupleKey2":"TupleValue2"
  * ```
+ * For call which do not pass contents to [tuples], [message] contents will be escaped and returned.
  */
 fun semiFormattedTuples(message: String, vararg tuples: Pair<String, String>): String {
     if (tuples.isEmpty()) {
-        return "\"$message\""
+        return StringEscapeUtils.escapeJson(message)
     }
     val formattedTuples = tuples.joinToString(
-            separator = "\", \"",
-            transform = { """${it.first}":"${StringEscapeUtils.escapeJson(it.second)}""" })
-    return """"${StringEscapeUtils.escapeJson(message)}", "$formattedTuples""""
+            separator = ", ",

```

Comment:
Why does this no longer need escaping?

Reply:
Previously it was this: `"\", \""` I have now moved the escaped double quotes into the `transform` function so this string no longer needs any escaping in.

Reply:
I thought so, just couldn't get my head around it all :)